tests/page_test.go

```diff
@@ -62,6 +69,7 @@ func TestNestedFrames(t *testing.T) {
 	assert.NotNil(t, button1Handle)
 	err = button1Handle.Click(nil)
 	assert.Nil(t, err)
+	tb.Browser.Close()
```

Comment:
Is there a reason why the closing of the browser isn't deferred?

Reply:
No, there isn't. I fixed it.